sdk/core/Azure.Core.TestFramework/src/IgnoreServiceErrorAttribute.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+
+namespace Azure.Core.TestFramework
+{
+    /// <summary>
+    /// Marks tests failing with specific service error codes as "inconclusive".
+    /// </summary>
+    /// <remarks>
+    /// This is intended for use with intermittent service issues that happen only during internal testing,
+    /// or that are temporary issues causing test failures but will be fixed. Please engage with service teams
+    /// before using this attribute to see if live test issues can be resolved first.
+    /// </remarks>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true, Inherited = true)]
+    public class IgnoreServiceErrorAttribute : Attribute
+    {
+        private readonly string _statusMessage;
+        private readonly string _errorCodeMessage;
+
+        /// <summary>
+        /// Creates an instance of the <see cref="IgnoreServiceErrorAttribute"/> class.
+        /// </summary>
+        /// <param name="status">The HTTP status code to ignore e.g., 400.</param>
+        /// <param name="errorCode">The Azure error code to ignore e.g., BadParameter.</param>
+        public IgnoreServiceErrorAttribute(int status, string errorCode)
+        {
+            _statusMessage = $"Status: {status.ToString(CultureInfo.InvariantCulture)}";
```

Comment:
Should we throw if status or errorCode are null/empty?

Reply:
Could. I figured for an internal-ish attribute not necessary, but it wouldn't hurt.